### ISSUE TITLE
This is not an issue: Your work is 🔥🔥🔥

### ISSUE THREAD

**Issue by EhuddR** [chetachiezikeuzor/Writr-Goals#1]
Hello!
 
I just want to say I'm impressed. woooow! You're taking Obsidian to the next level.  
As a social scientist this makes the use of obsidian a lot fun and interactive just what I was waiting for.

Keep up the good work. And greetings from Ecuador.

**Comment by chetachiezikeuzor** [comment]
Thank you so much! The plugin is not yet ready to be used but I hope to have it done and explain its functions. The plan is to allow the user to set up writing goals per writing project (note). The side status board will give you a visual to better understand your writing progression based on the goals you've set. I want it to be fairly similar to what Ulysses has to offer, which is what I and my siblings have been missing with using Obsidian. Although I'm not yet done with the plugin, I'd love to get some input on what would make this a good plugin in building a writing environment in Obsidian, so feel free to let me know your thoughts!

**Comment by EhuddR** [comment]
Yes, I fully understood the basic functionality of the plugin just by watching the picture you posted.
Of course, I will be happy to contribute once I have tested the plugin. I can't wait.

**Comment by chetachiezikeuzor** [comment]
Awesome! I looks forward to your input 🤗
